### PR TITLE
Update assessment to include SQL Refinement

### PR DESCRIPTION
**Pull request by sarahmathey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+
+
+
+### Exercise One: SQL Refinement
+**Objective**: Refine [this SQL Query](https://docs.google.com/document/d/1idPKfrhwhuw5C5qDMwWun0MI5wQaBI288RFMafcTDos/edit?usp=sharing) so that it is easily readable by your teammates and fix any syntax errors you come across. 
+
+**Background**: 
+  - We store our data in a redshift database, and while we’ve done a lot of good work modeling this raw resource to help ourselves and teams across the organization quickly answer basic questions about fundraising patterns on our site through a BI tool, many of the deeper explorations our team does for the organization starts with manipulating the data stored in our redshift cluster using SQL. 
+  - This query is a messy version of the query we used to pull the data used in exercise two of this take home. All of the where clauses, and table sources are correct and don’t need to be updated, but this is obviously not a great work product to share with colleagues - it’s hard to scan for quick understanding of the work, uses CTEs and subqueries inconsistently, and there are some syntactical errors that would cause this query run to fail.
```
Comment (eschick36) [inline README.md]: Is it too obvious to say "uses CTEs and subqueries inconsistently"? Maybe we just say "has inconsistent syntax"?

**Review by eschick36 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
-## Data Scientist Technical Assessment
+# Data Scientist Technical Assessment
 
+## Expectations
+
+1. We don’t expect you to spend more than 90 minutes on these two exercises, so just work through the questions within the time limit and see what you complete 
+2. In the interest of respecting your free time, we suggest limiting any packages or modules imported to those referenced in the job description
+3. We’re not looking for award winning visualizations, if you chose to generate a visualization feel free to use the default plot style
```
Comment (dstern04) [inline README.md:7]:
```suggestion
3. We’re not looking for award winning visualizations, if you chose to generate a visualization feel free to use the default plot style
```
```suggestion
3. We’re not looking for award-winning visualizations, if you chose to generate a visualization feel free to use the default plot style
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+4. We think the provided data is sufficient to answer some really interesting questions and do not expect you to incorporate any additional data sources 
+5. We don’t expect you to be familiar with our filing data - please make assumptions, but be sure to document them in your code so that we can follow your thought process.
+6. When you’re ready, send us back a text file with the refined SQL query (for exercise 1) and a markdown file with your work (either RMarkdown or Jupyter notebook) or an R or Python file with your code  + comments to help us follow your work (for exercise 2)
+
+
+
+### Exercise One: SQL Refinement
+**Objective**: Refine [this SQL Query](https://docs.google.com/document/d/1idPKfrhwhuw5C5qDMwWun0MI5wQaBI288RFMafcTDos/edit?usp=sharing) so that it is easily readable by your teammates and fix any syntax errors you come across.
```
Comment (dstern04) [inline README.md]: thoughts on "refine" vs "refactor"?

**Review by dstern04 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dstern04) [inline README.md]:
```suggestion
3. We’re not looking for award-winning visualizations, if you chose to generate a visualization feel free to use the default plot style
```

File: README.md (hunk 3 above)
Comment (eschick36) [reply README.md]: I like refactor!

**Review by eschick36 — APPROVED** [review]

**Review by dstern04 — APPROVED** [review]

**Pull request merged**